Deprecate repository (#27)
DevExpress/dashboard-extensions: readme.md

> ⛔ DEPRECATED. This repository was deprecated. To get an up-to-date custom items, refer to the following examples:

Reviewer: ```suggestion
⛔ DEPRECATED. This repository was deprecated. For up-to-date information on custom items, refer to the following examples:
```

* [Dashboard for ASP.NET Core - Custom Item Gallery](https://github.com/DevExpress-Examples/asp-net-core-dashboard-custom-item-gallery)
* [Dashboard for Angular - Custom Item Gallery](https://github.com/DevExpress-Examples/dashboard-angular-app-custom-item-gallery)
* [Dashboard for React - Custom Item Gallery](https://github.com/DevExpress-Examples/dashboard-react-app-custom-item-gallery)

For a custom panel that displays a list of available dashboards, visit the following examples:

* [Dashboard for ASP.NET Core - How to Implement a Custom Service and UI for Managing Dashboards List](https://github.com/DevExpress-Examples/asp-net-core-dashboard-custom-panel)
* [Dashboard for Angular - How to Implement a Custom Service and UI for Managing Dashboards List](https://github.com/DevExpress-Examples/angular-dashboard-custom-panel)

The current repository will not be updated in the future.

## DevExpress Dashboard Extensions

> 

Reviewer: Please update the following lines below, because I cannot access them.

Lines 97 and 137:
Can we update these identical lines as follows?

`> Note that the Dashboard Panel extension consists of HTML, JavaScript, and CSS files, and requires additional steps to integrate into the Web Dashboard. See the [Dashboard Panel](docs/dashboard-panel.md) section for details.`

Line 143:
`You can use a code of these extensions as a base for your own dashboard item extension [development](https://docs.devexpress.com/Dashboard/117546).`

Please change "a code" to "the code".

Line 168:
`6. Edit extension's code in the **src** folder and save your changes.`

Please change "Edit extension's code" to "Edit the extension's code".

Line 184:
Can you update this line as follows?

`To address questions regarding the Web Dashboard and JavaScript API, use the [DevExpress Support Center](https://www.devexpress.com/Support/Center).`

Reviewer 2: Oh, I missed that the part you've mentioned should be removed. Thanks, Albert.

Reviewer: Oh, I didn't know you were going to remove those parts. Okay, thank you.

> This repository contains [dashboard extensions](https://docs.devexpress.com/Dashboard/117232) that allow you to add the additional functionality to the Web Dashboard. 

Reviewer: ```suggestion
This repository contains [dashboard extensions](https://docs.devexpress.com/Dashboard/117232) that allow you to add additional functionality to the Web Dashboard.
```
